Makefile

```diff
@@ -57,6 +57,9 @@ BUILDTAGS ?= \
 	$(shell hack/libsubid_tag.sh) \
 	exclude_graphdriver_devicemapper \
 	seccomp
+BUILDTAGS ?= \
+	$(BUILDTAGS_NO_ROOTLESS) \
+	native_rootless
```

Comment:
IMO the build tag should be opt out and not opt in.
If we make this opt in we all distributions or people who manually set builds tags would need to update them to keep already working functionality. This would even require a update to the fedora spec.

Reply:
Yes, this must be inverted.  I am still fighting it not to hang ... no clear solution yet